agent/consul/leader_metrics.go

```diff
@@ -3,6 +3,7 @@ package consul
 import (
 	"context"
 	"fmt"
+	"github.com/hashicorp/consul/agent/connect"
```

Comment:
Do you use `goimports` to fix the imports? 

I have GoLand configured with a File Watcher that runs `goimports -w -local github.com/hashicorp/consul` so that it formats the imports in the correct order. It's not perfect, but it works most of the time. We could also look at one of the newer tools (ex: https://github.com/daixiang0/gci) which might be more reliable.

I think this import should be in the third block of imports.

Reply:
I added `goimports` as a watcher, let me see how reliable it is before trying another tool.